source/plugin/items/usage.rst

```diff
@@ -55,17 +55,17 @@ value to ``true`` to imply that the item will never break. All of this is enclos
 ``ItemStack`` to return our changes back to the ``ItemStack``.
 
 Different keys will require different values based on their job. For example, to change the lore of an item, one would
-need to specify a ``List`` of :javadoc:`Text` rather than an boolean or other value. It is also important to perform
+need to specify a ``List`` of :javadoc:`Component` rather than an boolean or other value. It is also important to perform
```

Comment:
"an boolean"? idk how or when that crept in. Can you change it to "a boolean" please?